micro_ros_diagnostic_updater/include/micro_ros_diagnostic_updater/micro_ros_diagnostic_updater.h

```diff
@@ -21,6 +21,9 @@
 #include <micro_ros_diagnostic_msgs/msg/micro_ros_diagnostic_status.h>
 
 #define MICRO_ROS_UPDATER_MAX_NUMBER_OF_TASKS 5
+#define MICRO_ROS_UPDATER_MAX_NUMBER_OF_TASKS 5
+
+static const char UROS_DIAGNOSTICS_UPDATER_TOPIC[] = "/diagnostics_uros";
```

Comment:
I think we should definitely keep "hardcoded" the `diagnostics_uros` topic. However, we should remove the trailing `/` to allow it to attach to existent namespaces.
Then via config.h.in (like on the bridge). Provide any other text that should be preprended to the diagnostics_uros topic name. Making it
#ifdef SOMETHING
@something@/diagnostic_uros
#endif
Or alike

Reply:
if we just allow to _prefix_ the output topic, we should also only allow to _prefix_ the bridge input topic.
also the bridge output topic should be fixes, since it is [hard-coded in the diagnostic aggregator as well](https://github.com/ros/diagnostics/blob/ros2-devel/diagnostic_aggregator/src/aggregator.cpp#L104).